### PR TITLE
Fix duplicated CadShape mesh load and improve warnings

### PR DESCRIPTION
**Pull request by stefaniapedrazzi**
Address #5356 and ##5313:
* [x] Add the same warning about complex mesh with more than 100'000 vertices in the Mesh node
* [x] Add warning in CadShape node when the imported mesh is fully transparent -> otherwise users may think that the import is buggy
* [x] Avoid creating WREN objects twice when finalizing the CadShape node.

### CODE REVIEW

File: docs/reference/changelog-r2022.md [hunk 1]
```diff
@@ -12,6 +12,9 @@
     - Fixed close and resize buttons rendered in black in the [RangeFinder](rangefinder.md) overlay if underlying pixel value is `inf` ([#5337](https://github.com/cyberbotics/webots/pull/5337)).
     - Fixed error message about missing PROTO declaration when copying and pasting a PROTO node that was just added using the Add Node dialog ([#5341](https://github.com/cyberbotics/webots/pull/5341)).
     - Fixed the description of base nodes in the "Add node" dialog ([#5346](https://github.com/cyberbotics/webots/pull/5346)).
+  - Enhancements
+    - Added warnings if external mesh used in [CadShape](cadshape.md) node is fully transparent ([#5359](https://github.com/cyberbotics/webots/pull/5359)).
+    - Added warning if external mesh used in [Mesh](mesh.md) node has more than 100'000 vertices ([#5359](https://github.com/cyberbotics/webots/pull/5359)).
```
Comment (BenjaminDeleze) [inline docs/reference/changelog-r2022.md:17]:
```suggestion
    - Added warning if external mesh used in [CadShape](cadshape.md) node is fully transparent ([#5359](https://github.com/cyberbotics/webots/pull/5359)).
    - Added warning if external mesh used in [Mesh](mesh.md) node has more than 100'000 vertices ([#5359](https://github.com/cyberbotics/webots/pull/5359)).
```

**Review by BenjaminDeleze — CHANGES_REQUESTED** [review]
Just a small remark, otherwise it seems good to me!

**Review by BenjaminDeleze — APPROVED** [review]
Thank you

**Review by BenjaminDeleze — APPROVED** [review]

**Comment by stefaniapedrazzi** [comment]
I just fixed a conflict in the ChangeLog.

**Review by BenjaminDeleze — APPROVED** [review]

**Pull request merged**